mteb/tasks/Retrieval/multilingual/PosIRRetrieval.py

```diff
@@ -0,0 +1,591 @@
+            "This test set is based on the computer_programming_code subset of the BAAI/IndustryCorpus2 dataset. "
+            + _COMMON_DESCRIPTION_SUFFIX
+        ),
+    )
+
+    load_data = _common_load_data
+
+
+class PosIRCurrentAffairsGovernmentAdministrationRetrieval(
+    MultilingualTask, AbsTaskRetrieval
+):
+    _industry_domain = "current_affairs_government_administration"
+    metadata = TaskMetadata(
+        **_POSIR_METADATA,
+        name="PosIRCurrentAffairsGovernmentAdministrationRetrieval",
+        domains=["News"],
+        description=(
+            "This test set is based on the current_affairs_government_administration subset of the BAAI/IndustryCorpus2 dataset. "
+            + _COMMON_DESCRIPTION_SUFFIX
+        ),
+    )
+
+    load_data = _common_load_data
+
+
+class PosIRElectricPowerEnergyRetrieval(MultilingualTask, AbsTaskRetrieval):
+    _industry_domain = "electric_power_energy"
+    metadata = TaskMetadata(
+        **_POSIR_METADATA,
+        name="PosIRElectricPowerEnergyRetrieval",
+        domains=["Engineering"],
+        description=(
+            "This test set is based on the electric_power_energy subset of the BAAI/IndustryCorpus2 dataset. "
+            + _COMMON_DESCRIPTION_SUFFIX
+        ),
+    )
+
+    load_data = _common_load_data
+
+
+class PosIRFilmEntertainmentRetrieval(MultilingualTask, AbsTaskRetrieval):
+    _industry_domain = "film_entertainment"
+    metadata = TaskMetadata(
+        **_POSIR_METADATA,
+        name="PosIRFilmEntertainmentRetrieval",
+        domains=["Entertainment"],
+        description=(
+            "This test set is based on the film_entertainment subset of the BAAI/IndustryCorpus2 dataset. "
+            + _COMMON_DESCRIPTION_SUFFIX
+        ),
+    )
+
+    load_data = _common_load_data
+
+
+class PosIRFinanceEconomicsRetrieval(MultilingualTask, AbsTaskRetrieval):
+    _industry_domain = "finance_economics"
+    metadata = TaskMetadata(
+        **_POSIR_METADATA,
+        name="PosIRFinanceEconomicsRetrieval",
+        domains=["Financial"],
+        description=(
+            "This test set is based on the finance_economics subset of the BAAI/IndustryCorpus2 dataset. "
+            + _COMMON_DESCRIPTION_SUFFIX
+        ),
+    )
+
+    load_data = _common_load_data
+
+
+class PosIRFinewebRetrieval(MultilingualTask, AbsTaskRetrieval):
+    _industry_domain = "fineweb"
+    metadata = TaskMetadata(
+        **_POSIR_METADATA,
+        name="PosIRFinewebRetrieval",
+        domains=["Web"],
+        description=(
+            "This test set is based on the fineweb dataset. "
+            + _COMMON_DESCRIPTION_SUFFIX
+        ),
+    )
+
+    load_data = _common_load_data
+
+
+class PosIRFireSafetyFoodSafetyRetrieval(MultilingualTask, AbsTaskRetrieval):
+    _industry_domain = "fire_safety_food_safety"
+    metadata = TaskMetadata(
+        **_POSIR_METADATA,
+        name="PosIRFireSafetyFoodSafetyRetrieval",
+        domains=["Non-fiction"],
```

Comment:
can you add non-fiction to the cases where it applied. I also seem like you can add "Written" for all datasets